src/tests/features/nftables_flowtable.at

```diff
@@ -6,6 +6,10 @@ AT_SKIP_IF([! NS_CMD([command -v iperf3 >/dev/null 2>&1])])
 AT_SKIP_IF([! NS_CMD([command -v sysctl >/dev/null 2>&1])])
 AT_SKIP_IF([! NS_CMD([command -v jq >/dev/null 2>&1])])
 
+dnl I don't understand why, tbut this test fails in fw-in-container.
```

Comment:
Same as other comments. Just call `SKIP_IF_FW_IN_CONTAINER()` at the start of the test.